src/Avalonia.Controls/ProgressBar.cs

```diff
@@ -105,21 +105,21 @@ private void ValueChanged(AvaloniaPropertyChangedEventArgs e)
         private class IndeterminateAnimation : IDisposable
         {
             private WeakReference<ProgressBar> _progressBar;
-            private IDisposable _indeterminateBindSubscription;
-            private TimeSpan _startTime;
+            //private IDisposable _indeterminateBindSubscription;
```

Comment:
Is it possible to do something with this before merge, its preferable not to have commented code in master.